### ISSUE TITLE
CheckBox and ChoiceFieldGroup: Incorrect state read by JAWS

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-core#645]
Here is the bug report:

Navigate to this page: Components/ChoiceField/index.html
Using TAB navigate to the third checkbox
It reads "Checkbox selected, checkboxes unchecked"
It should have read "Checkbox selected, checkbox checked".

Once I pressed spacebar to uncheck and recheck it fixed itself but not on first load.

Same behavior happens for radio buttons

**Comment by mikewheaton** [comment]
This issue was moved to OfficeDev/office-ui-fabric-js#44
